index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
 
-<form >
-    <input class="text-inp" type="text" placeholder=" Text Input">
-
-    <input class="email" type="text" placeholder=" Jeremy@gmail.com">
-
-    <textarea class="send-text" cols="30" rows="10" placeholder="
-Enter text here"></textarea>
-</form>
-
-<button type="button" class="btn-sec"> See Project </button>
-
-<div class="cont-line">.</div>
-
-
+    <form action="https://formspree.io/f/mlezygyr" method="post">
+        <ul class="input-list">
```

Comment:
- [ ] Do we need an `<ul>` tag here? The form input elements are block level elements and they can be style using CSS.